src/main/java/net/minecraftforge/common/config/IFieldWrapper.java

```diff
@@ -61,17 +61,21 @@
      */
     boolean handlesKey(String key);
 
-    /**
+    /**TODO remove both methods in future
```

Comment:
Why should we remove both methods? We need one to do the setup. We currently have two to maintain binary compatibility. In the future, we need probably only one, but zero is one to few.

Reply:
This whole system is dead in 1.13 with the advent of the new TOML-based configuration system. So any "remove in future" comments are pointless in their entirety, this code doesn't even exist in the 1.13 codebase.

Reply:
alright, I don't know what to do here anymore

Reply:
@tterrag1098 I designed this system to able to handle different backing configuration files without changing the interface to the mods. Therefore this infrastructure should be able to remain and maintain backwards comaptibility.